doc/classes/BackBufferCopy.xml

```diff
@@ -46,8 +46,10 @@
 	</methods>
 	<members>
 		<member name="copy_mode" type="int" setter="set_copy_mode" getter="get_copy_mode" enum="BackBufferCopy.CopyMode">
+			The copy mode of the BackBufferCopy (see constants).
```

Comment:
It's not generally useful to make the description a repetition of the member name. You need to describe what copy_mode is *for*.

Reply:
@cbscribe Any suggestions? This class is very small and I'm trying to come up with a description that doesn't just repeat the documentation for the `COPY_MODE_*` constants.

Reply:
It doesn't have to be complex, how about:

    Buffer mode. See [code]COPY_MODE_*[/code] constants.

But I would also suggest improving the constant descriptions.  They shouldn't say "Sets..." because they don't set things, they describe the options.